docs/docs/classic-ml/deep-learning/index.mdx

```diff
@@ -3,420 +3,124 @@ sidebar_position: 0
-    mlflow.log_metric("best_accuracy", study.best_value)
-```
-
-</details>
-
-<details>
-  <summary>Transfer Learning Workflows</summary>
-
-MLflow helps organize transfer learning and fine-tuning workflows:
-
-- **Base Model Registry**: Maintain a catalog of pre-trained models
-- **Fine-Tuning Tracking**: Monitor performance as you adapt models to new tasks
-- **Layer Freezing Analysis**: Compare different layer freezing strategies
-- **Learning Rate Scheduling**: Track the impact of different learning rate strategies for fine-tuning
-
-Example tracking a fine-tuning run:
-
-```python
-import mlflow
-import torch
-from transformers import AutoModelForSequenceClassification
-
-with mlflow.start_run():
-    # Log base model information
-    base_model_name = "bert-base-uncased"
-    mlflow.log_param("base_model", base_model_name)
-
-    # Create and customize model for fine-tuning
-    model = AutoModelForSequenceClassification.from_pretrained(base_model_name)
-
-    # Log which layers are frozen
-    frozen_layers = ["embeddings", "encoder.layer.0", "encoder.layer.1"]
-    mlflow.log_param("frozen_layers", frozen_layers)
-
-    # Freeze specified layers
-    for name, param in model.named_parameters():
-        if any(layer in name for layer in frozen_layers):
-            param.requires_grad = False
-
-    # Log trainable parameter count
-    trainable_params = sum(p.numel() for p in model.parameters() if p.requires_grad)
-    total_params = sum(p.numel() for p in model.parameters())
-    mlflow.log_params(
-        {
-            "trainable_params": trainable_params,
-            "total_params": total_params,
-            "trainable_percentage": trainable_params / total_params,
-        }
-    )
-
-    # Fine-tune and track results...
-```
-
-</details>
+MLflow provides comprehensive experiment tracking, model management, and deployment capabilities for deep learning workflows. From PyTorch training loops to TensorFlow models, MLflow streamlines your path from experimentation to production.
+
+## Get Started
+
+<TilesGrid>
+  <TileCard
+    image="/images/logos/pytorch-logo.svg"
+    title="PyTorch Guide"
+    description="Track custom PyTorch training loops with automatic metric and artifact logging."
+    href="/ml/deep-learning/pytorch"
+    linkText="View guide →"
+    containerHeight={64}
+  />
+  <TileCard
+    image="/images/logos/TensorFlow-logo.svg"
+    title="TensorFlow Guide"
+    description="Integrate TensorFlow models with autologging and TensorBoard visualization support."
+    href="/ml/deep-learning/tensorflow"
+    linkText="View guide →"
+    containerHeight={64}
+  />
+  <TileCard
+    image="/images/logos/keras-logo.svg"
+    title="Keras Guide"
+    description="Use Keras 3.0 multi-backend capabilities with unified MLflow tracking."
+    href="/ml/deep-learning/keras"
+    linkText="View guide →"
+    containerHeight={64}
+  />
+  <TileCard
+    image="/images/logos/spacy-logo.svg"
+    title="spaCy Guide"
```

Comment:
Not sure if we still want to highlight spacy. It's dead

<img width="913" height="450" alt="image" src="https://github.com/user-attachments/assets/8c77a0f8-e365-4fce-b56d-56285503acb4" />

Reply:
Yeah I'll handle this in a separate PR (delisting it as a 'named' flavor and removing it from the tile card entry)